data-scripts/create-dummy-data.mjs

```diff
@@ -109,10 +166,18 @@ function getRandomInt(min, max) {
 let firstNameIndex = 0;
 
 async function createHousehold(lastName) {
+const randomAddress1Index = Math.floor(Math.random()*addresses1.length);
```

Comment:
instead of doing this math inline call the getRandomInt function 

const randomAddress1Index = getRandomInt(0, addresses1.length);

Reply:
fixed